### PR TITLE
[sdk] fix: test vectors missing for mosaic revocation, receipt and detached cosignature

### PR DESCRIPTION
**Pull request by Wayonb**
problem: test vectors are missing for mosaic revocation, receipt, and detached cosignature
solution: add test vectors for mosaic revocation, receipt, and detached cosignature

### CODE REVIEW

File: sdk/javascript/package-lock.json
Comment (Jaguar0625) [inline sdk/javascript/package-lock.json:1]: seems like this is unrelated change

File: sdk/python/testvectors/symbol/detached_cosignature.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from binascii import unhexlify
+
+from symbolchain.sc import DetachedCosignature, Hash256, PublicKey, Signature
+
+
+def create_detached_cosignature():
+	cosignature = DetachedCosignature()
+	cosignature.version = 0
+	cosignature.signer_public_key = PublicKey('67fa12789f80766d329c7f687c5c5f889a82f5e9c3e7996ae4ffe48c34299de7')
```
Comment (Jaguar0625) [inline sdk/python/testvectors/symbol/detached_cosignature.py:9]: please capitalize hex strings everywhere

File: sdk/python/testvectors/__main__.py [hunk 2]
```diff
@@ -176,7 +176,7 @@ def create_block(self, test_name, original_descriptor):
 	@staticmethod
 	def create_manually(test_name, original_descriptor):
 		del test_name
-		return original_descriptor['object'], None
+		return original_descriptor['object'], str(original_descriptor['object'])
```
Comment (Jaguar0625) [inline sdk/python/testvectors/__main__.py:179]: do you need str here ?

File: sdk/python/testvectors/symbol/mosaic_recovation.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+SAMPLE_ADDRESS_ALIAS = 'TFIXOYLI2JBFPWAAAAAAAAAAAAAAAAAAAAAAAAA'
```
Comment (Jaguar0625) [inline sdk/python/testvectors/symbol/mosaic_recovation.py:1]: mosaic_supply_revocation filename ?

File: sdk/javascript/package-lock.json
Comment (Wayonb) [reply sdk/javascript/package-lock.json:1]: Yeah, after building, I noticed the lock was not updated.  can move it if you want?

File: sdk/python/testvectors/__main__.py (hunk 2 above)
Comment (Wayonb) [reply sdk/python/testvectors/__main__.py:179]: Probably not but let me check

Comment (Wayonb) [reply sdk/python/testvectors/__main__.py:179]: I left the str() or else it fails trying to convert to JSON with ``TypeError: Object of type DetachedCosignature is not JSON serializable``

**Review by Jaguar0625 — APPROVED** [review]
👍
